addons/auth_totp_mail_enforce/models/res_users.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import babel.dates
+import logging
+import re
+
+from datetime import datetime, timedelta
+
+from odoo import _, models
+from odoo.exceptions import AccessDenied
+from odoo.http import request
+from odoo.tools.misc import babel_locale_parse
+
+from odoo.addons.auth_totp.models.totp import hotp, TOTP
+
+_logger = logging.getLogger(__name__)
+
+TOTP_RATE_LIMITS = {
```

Comment:
Why do we not making this configurable in an `ir.config_parameter`?

Reply:
because it's premature, and we can always do that later if we feel there is a need?